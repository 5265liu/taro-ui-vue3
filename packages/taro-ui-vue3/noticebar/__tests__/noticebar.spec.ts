import { mount } from '@vue/test-utils'
import AtNoticebar from '../src/index.vue'

const AXIOM = 'Rem is the best girl'

describe('AtNoticebar.vue', () => {
  test('render test', () => {
    const wrapper = mount(AtNoticebar, {
      slots: {
        default: AXIOM,
      },
    })
    expect(wrapper.text()).toEqual(AXIOM)
  })
})
