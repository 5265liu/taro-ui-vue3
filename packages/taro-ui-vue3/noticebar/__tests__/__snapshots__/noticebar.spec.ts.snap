// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AtNoticebar Snap render AtNoticebar -- props icon 1`] = `
<view
  class="at-noticebar"
>
  <!-- close icon -->
  <!--v-if-->
  <!-- content -->
  <view
    class="at-noticebar__content"
  >
    <!-- icon -->
    <view
      class="at-noticebar__content-icon"
    >
      <text
        class="at-icon at-icon-volume-plus"
      />
    </view>
    <!-- text -->
    <view
      class="at-noticebar__content-text"
    >
      <!-- default content slot -->
      <view
        animation="[object Object]"
        class="at-noticebar__content-inner"
        id="J_fibw"
      >
        
        这是内容
        
      </view>
      <!-- show more content -->
      <!--v-if-->
    </view>
  </view>
</view>
`;

exports[`AtNoticebar Snap render AtNoticebar -- props moreText & showMore 1`] = `
<view
  class="at-noticebar at-noticebar--single"
>
  <!-- close icon -->
  <!--v-if-->
  <!-- content -->
  <view
    class="at-noticebar__content"
  >
    <!-- icon -->
    <!--v-if-->
    <!-- text -->
    <view
      class="at-noticebar__content-text"
    >
      <!-- default content slot -->
      <view
        animation="[object Object]"
        class="at-noticebar__content-inner"
        id="J_fafs"
      >
        
        这是内容
        
      </view>
      <!-- show more content -->
      <view
        class="at-noticebar__more"
      >
        <text
          class="text"
        >
          查看更多
        </text>
        <view
          class="at-noticebar__more-icon"
        >
          <text
            class="at-icon at-icon-chevron-right"
          />
        </view>
      </view>
    </view>
  </view>
</view>
`;

exports[`AtNoticebar Snap render AtNoticebar -- props show 1`] = `
<view
  class="at-noticebar"
>
  <!-- close icon -->
  <view
    class="at-noticebar__close"
  >
    <text
      class="at-icon at-icon-close"
    />
  </view>
  <!-- content -->
  <view
    class="at-noticebar__content"
  >
    <!-- icon -->
    <!--v-if-->
    <!-- text -->
    <view
      class="at-noticebar__content-text"
    >
      <!-- default content slot -->
      <view
        animation="[object Object]"
        class="at-noticebar__content-inner"
        id="J_7nr0"
      >
        
        这是内容
        
      </view>
      <!-- show more content -->
      <!--v-if-->
    </view>
  </view>
</view>
`;

exports[`AtNoticebar Snap render AtNoticebar -- props single 1`] = `
<view
  class="at-noticebar at-noticebar--single"
>
  <!-- close icon -->
  <!--v-if-->
  <!-- content -->
  <view
    class="at-noticebar__content"
  >
    <!-- icon -->
    <!--v-if-->
    <!-- text -->
    <view
      class="at-noticebar__content-text"
    >
      <!-- default content slot -->
      <view
        animation="[object Object]"
        class="at-noticebar__content-inner"
        id="J_bc46"
      >
        
        这是内容
        
      </view>
      <!-- show more content -->
      <!--v-if-->
    </view>
  </view>
</view>
`;

exports[`AtNoticebar Snap render AtNoticebar -- props speed 1`] = `
<view
  class="at-noticebar"
>
  <!-- close icon -->
  <!--v-if-->
  <!-- content -->
  <view
    class="at-noticebar__content"
  >
    <!-- icon -->
    <!--v-if-->
    <!-- text -->
    <view
      class="at-noticebar__content-text"
    >
      <!-- default content slot -->
      <view
        animation="[object Object]"
        class="at-noticebar__content-inner"
        id="J_e1uj"
      >
        
        这是内容
        
      </view>
      <!-- show more content -->
      <!--v-if-->
    </view>
  </view>
</view>
`;
